README.md

```diff
@@ -12,7 +12,8 @@ This web application is built to display covid-19 data of countries affected by
 
 ## Live Demo
 
-[View Life](https://duls-travelers-hub.herokuapp.com/)
+- [View video](https://www.loom.com/share/2f173a6ca1594c60a9e760d6d6ef74a3)
+- [View Demo](https://duls-covid-reporter.herokuapp.com/)
   
```

Comment:
Your _ReadMe_ needs some improvements so as to make it appear more professional. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this project,  which will likely feature on your portfolio. Kindly ensure to improve upon these aspects of your ReadMe. 
- [ ] How to set up, run and test this project should be detailed, clear, and well organized.
- [ ] The code aspects in each procedure should be in backticks`.